desktop/core/src/desktop/auth/api_authentications.py

```diff
@@ -125,6 +127,30 @@ def _handle_public_key(self, access_token):
 
             return public_key_pem
 
+  def _handle_jku_ha(self):
+    res = None
+
+    key_server_urls = AUTH.JWT.KEY_SERVER_URL.get()
+    if not key_server_urls:
+      return None
+
+    if "," in key_server_urls:
+      # Config format is "['url1', 'url2']" for HA, so we need to clean up and split correctly in list.
+      key_server_urls_list = key_server_urls.replace("%20", "").replace("['", "").replace("']", "").replace("'", "").split(',')
+
+      for jku in key_server_urls_list:
+        try:
+          res = requests.get(jku.rstrip('/'), verify=False)
```

Comment:
you might be able to do just `requests.head` if headers are all you are interested in.

Reply:
@amitsrivastava Is there any advantage of using HEAD instead of GET here? I tried both and latency-wise both were returning in almost similar time.

Reply:
It depends. If the request anyways is super light, then there isn't a difference. Sometimes, the servers may decide not to generate the body for a HEAD only request. I think, in our Hue servers, if you make a HEAD request for the excel-export of query results, server will skip the actual conversion to excel. But, your case is probably anyways the same either ways.

Reply:
Ahh I see, good point to note!
Yes I was not getting the response content but they both were taking similar time so we can use any I feel.